src/main/java/com/czertainly/api/model/core/certificate/CertificateChainDownloadResponseDto.java

```diff
@@ -2,12 +2,12 @@
 
 import io.swagger.v3.oas.annotations.media.Schema;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 
 import java.util.List;
 
 @Data
-public class CertificateChainDownloadResponseDto {
-
+public class CertificateChainDownloadResponseDto extends CertificateDownloadResponseDto {
```

Comment:
If extending from CertificateDownloadResponseDto, there are properties that are duplicate. Only `completeChain` is additional